doc/bluechi-examples/golang/list_nodes.go

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
The `doc/bluechi-examples` is intended for the bindings (only python currently) - maybe the directory name is misleading. Could you please move it to `doc/api-examples/go/list_nodes.go`?